ui/app/AppLayouts/Browser/popups/BrowserConnectionModal.qml

```diff
@@ -31,9 +31,11 @@ StatusModal {
 
     function postMessage(isAllowed){
         interactedWith = true
-        request.isAllowed = isAllowed;
         RootStore.currentTabConnected = isAllowed
-        web3Response(Web3ProviderStore.web3ProviderInst.postMessage(Constants.api_request, JSON.stringify(request)))
+        if(isAllowed){
+            Web3ProviderStore.addPermission(request.hostname, request.permission)
```

Comment:
I wonder, should it be the opposite? We add a permission if it's not allowed? Or maybe I'm misunderstanding those lines and the addPermission happens because it is allowed and the line below is what asks for the permission or send the result of it being blocked?

Reply:
@jrainville see [here](https://github.com/status-im/status-desktop/blob/f5d53f9112cc2f1dbd14b31997d476d3936c0ecb/ui/app/AppLayouts/Browser/popups/BrowserConnectionModal.qml#L35-L37), that is how it is in the commit which was tested and approved in the "old" master, so I just adapted those changes here.

Reply:
Ok perfect.